core/templates/pages/contributor-dashboard-page/translation-language-selector/translation-language-selector.component.html

```diff
@@ -108,17 +109,22 @@
 
   .oppia-translation-language-selector-dropdown-option {
     color: rgb(104, 104, 104);
+    cursor: pointer;
     display: flex;
     flex: 1;
     margin-left: 16px;
     padding: 8px;
   }
-
-  .oppia-translation-language-selector-dropdown-option-selected {
+  .oppia-translation-language-selector-dropdown-option:hover {
     background-color: rgb(238, 238, 238);
     border-radius: 4px;
   }
 
+  .oppia-translation-language-selector-dropdown-option-selected {
+    background-color: rgb(215, 255, 237);
```

Comment:
Where did this color come from? I don't see it in our design guidelines: https://docs.google.com/document/d/1R9PWRBA7yPCiHxc325gx1xHd_RMLRlCjMFWDCowUPag/edit#heading=h.kwpmfhhieo37. 

cc @mschanteltc